apps/src/templates/ProtectedStatefulDiv.jsx

```diff
@@ -14,11 +15,16 @@ import Radium from 'radium'; // eslint-disable-line no-restricted-imports
 class ProtectedStatefulDiv extends React.Component {
   static propTypes = {
     contentFunction: PropTypes.func,
-    children: PropTypes.node
+    children: PropTypes.node,
+    canupdate: PropTypes.string
```

Comment:
Any reason we can't use a `bool` here?

Reply:
Originally, I had canUpdate: PropTypes.bool. However, I received the following warning:
react_devtools_backend.js:4026 Warning: React does not recognize the `canUpdate` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `canupdate` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Then I changed to all canUpdate to all lowercase and received this error:react_devtools_backend.js:4026 Warning: Received `false` for a non-boolean attribute `canupdate`. If you want to write it to the DOM, pass a string instead: canupdate="false" or canupdate={value.toString()}.
If you used to conditionally omit it with canupdate={condition && value}, pass canupdate={condition ? value : undefined} instead.

Also, the integration tests failed at this point.

So once I revised to how it is currently, integration tests passed. Reference: https://stackoverflow.com/questions/68741445/warning-react-does-not-recognize-the-warnkey-prop-on-a-dom-element

Reply:
Gotcha - I wonder if you just need to omit the prop [here](https://github.com/code-dot-org/code-dot-org/blob/6ec867e90a0c7c331e1ec480f9e015b3f59ae52b/apps/src/templates/ProtectedStatefulDiv.jsx#L46)? Sounds like React is complaining about the prop being set on the div, but I don't think the prop itself shouldn't cause any issues on the component.

Reply:
Oh! That's cool! Will change canUpdate back to bool.